content/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -178,4 +178,4 @@ After setting the dataset permission, connect to the share.
 ### Connecting to a Multiprotocol Share
 After creating and configuring the shares, connect to the mulit-protocol share using either SMB or NFS protocols from a variety of client operating systems including Windows, Apple, FreeBSD, and Linux/Unix systems.
 
-For more information on accessing shares, see [Mounting the SMB Share]({{< relref "/SCALETutorials/Shares/_index.md #mounting-the-smb-share" >}}) and [Connecting to the NFS Share]({{< relref "AddingNFSShares.md #connecting-to-the-nfs-share" >}}).
+For more information on accessing shares, see [Mounting the SMB Share]({{< relref "/SCALETutorials/Shares/_index.md#mounting-the-smb-share" >}}) and [Connecting to the NFS Share]({{< relref "AddingNFSShares.md#connecting-to-the-nfs-share" >}}).
```

Comment:
this link works, but does not make sense, it is not going to information on mounting the share. I wondered if a commit moving general content into the index.md file is missing but it is not there in the Master branch. This link to Mounting the SMB Share needs to get fixed. Sorry, I didn't catch it in the review of the Master branch PR.
```suggestion
For more information on accessing shares, see [Mounting the SMB Share]({{< relref "/SCALETutorials/Shares/_index.md#mounting-the-smb-share" >}}) and [Connecting to the NFS Share]({{< relref "AddingNFSShares.md#connecting-to-the-nfs-share" >}}).
```